src/systems/systems/src/shared/mod.rs

```diff
@@ -83,6 +83,11 @@ pub trait EngineUncorrectedN2 {
     fn uncorrected_n2(&self) -> Ratio;
 }
 
+pub trait CabinAltitude {
+    fn cabin_altitude(&self) -> Length;
+    fn cabin_pressure(&self) -> Pressure;
+}
```

Comment:
**thought**: I'm wondering if we can improve this name a bit. `CabinAltitude` with `cabin_altitude` and `cabin_pressure` has a few (minor) problems:
- "cabin" is repeated a lot.
- pressure is not initially expected to be in this type.

**suggestion**: Maybe the following would be a better option? Thoughts?

```suggestion
pub trait Cabin {
    fn altitude(&self) -> Length;
    fn pressure(&self) -> Pressure;
}
```

Reply:
I followed your suggestion and renamed the trait to `Cabin`.